Stepped tracker docs (#2291)
jpmorganchase/salt-ds: site/docs/components/stepped-tracker/usage.mdx

---
title:
  $ref: ./#/title
layout: DetailComponent
sidebar:
  exclude: true
data:
  $ref: ./#/data
---

### When to use Stepped Tracker

- If there are separate steps within a process such as a wizard or a multi-step form.

- Where a process can be broken into distinct, bite-sized sections that are less daunting than lengthy forms.

### When not to use Stepped Tracker

- If there are fewer than three steps in a process.

- As navigation; the steps are not interactable.

### Content

Stepped Tracker labels never truncate, only wrap. Therefore, ensure they are short and self-explanatory.

### Import

> To import Salt Provider from the lab Salt package (will be core eventually), use:

Reviewer: ```suggestion
To import Stepped Tracker from the lab Salt package (will be core eventually), use:
```

```
import { SteppedTracker, TrackerStep, StepLabel } from "@salt-ds/lab";
```

### Props

#### Stepped Tracker

<PropsTable packageName="lab" componentName="SteppedTracker" />

> #### TrackerStep

Reviewer: ```suggestion
#### Tracker Step
```

---
title:
  $ref: ./#/title
layout: DetailComponent
sidebar:
  exclude: true
data:
  $ref: ./#/data
---

### When to use Stepped Tracker

- If there are separate steps within a process such as a wizard or a multi-step form.

- Where a process can be broken into distinct, bite-sized sections that are less daunting than lengthy forms.

### When not to use Stepped Tracker

> - If there are less than three sequential steps in a process

Reviewer: ```suggestion
- If there are less than three sequential steps in a process.
```

nit

- As navigation; the steps are not interactable.

### Content

The Stepped Tracker labels will never truncate, only wrap. Therefore, they should be short and self-explanatory.

### Import

To import Salt Provider from the lab Salt package (will be core eventually), use:

```
import { SteppedTracker, TrackerStep, StepLabel } from "@salt-ds/lab";
```

### Props

#### SteppedTracker

<PropsTable packageName="lab" componentName="SteppedTracker" />

#### TrackerStep

<PropsTable packageName="lab" componentName="TrackerStep" />

#### StepLabel

> <PropsTable packageName="lab" componentName="StepLabel" />

Reviewer: Looks like there isn't any props used for StepLabel component - do we  still want to show it?
